tests/versioning_test.go

```diff
@@ -39,14 +39,14 @@ import (
 
 	enumspb "go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/serviceerror"
-	"go.temporal.io/api/workflowservice/v1"
+	workflowservice "go.temporal.io/api/workflowservice/v1"
```

Comment:
undo this? looks like an over-aggressive goimports

Reply:
The linter is not smart enough to know that if you have a variable named `v1` and an import line that ends with `v1` there's no shadowing.

Reply:
I'd rather fix/disable the broken linter than appease it